.github/workflows/sync-from-sunbeam.yaml

```diff
@@ -0,0 +1,46 @@
+# This workflow is to sync dashboards and alert rules from the Sunbeam repo and create a PR if there is an update.
+name: sync-from-sunbeam
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: '43 1 * * 1'
+
+
+env:
+  REPO_OWNER: canonical
+  REPO_PATH: openstack-exporter-operator
+
+jobs:
+
+  sync-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+          
+      - name: Pull dashboard and alert rules from sunbeam
+        run: |
+          bash scripts/sync-from-sunbeam.sh
+
+      - uses: actions/create-github-app-token@v1
+        id: app_token
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          owner: ${{ env.REPO_OWNER }}
+          private-key: ${{ secrets.APP_PEM }}
+          repositories: ${{ env.REPO_PATH }}
+      - name: Create pull request
```

Comment:
This should only create a PR if there are changes.  It looks like this will always create the PR?

Reply:
Well according to https://github.com/canonical/create-pull-request by default, the action fails if no local changes are detected. I have enabled `ignore-no-changes` option to make sure the action to do nothing and exit successfully when there are no changes.